docs/pywbemcli/cmdlineinterface.rst

```diff
@@ -175,7 +175,7 @@ user):
 
 .. code-block:: text
 
-    pywbemcli> [GENERAL-OPTIONS] [COMMAND-GROUP] COMMAND [COMMAND-OPTIONS] [ARGS]
+    pywbemcli> [GENERAL-OPTIONS] [COMMAND-GROUP] COMMAND [ARGS] [CMD-OPTIONS]
```

Comment:
`COMMAND-GROUP` is mandatory, so the square brackets around it should be removed.